pages/ColophSupplTit.xml

```diff
@@ -65,7 +65,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         are working on a manuscript record and we do not know the textual unit well enough, so we need to
         register as much as possible, but not overstate. Fortunately, all this different
         information can be collected together, and the use of the same set of attributes can
-        help. An additional reason for this is that this information can be joined to one 
+        help. An additional <att>reason</att> for this is that this information can be joined to one 
```

Comment:
Thank you for explaining the reason why `@reason` is not listed in the guidelines. But this markup here is not strictly correct, because the word "reason" in this sentence does not refer to an attribute. But if this an acceptable way for quickly having `@reason` on the list, I will not complain :)